src/frontend/state.ts

```diff
@@ -45,6 +45,7 @@ export const initialSearchParams: SearchParams = {
   publishingStatus: PublishingStatus.All,
   politicalStatus: PoliticalStatus.Any,
   government: '',
+  linksExactMatch: false,
```

Comment:
This state isn't being recorded in the URL, for some reason, which means that when a page of search results is refreshed, it does a new search that allows inexact matches.